src/Interfaces/AbstractPolyhedron_functions.jl

```diff
@@ -431,3 +432,50 @@ function plot_recipe(P::AbstractPolyhedron{N}, ε::N=zero(N)) where {N<:Real}
         return x, y
     end
 end
+
+"""
+    chebyshev_center(P::AbstractPolyhedron{N};
+                     [get_radius]::Bool=false,
+                     [backend]=default_polyhedra_backend(P, N),
+                     [solver]=JuMP.with_optimizer(GLPK.Optimizer)
+                     ) where {N<:AbstractFloat}
+
+Compute the [Chebyshev center](https://en.wikipedia.org/wiki/Chebyshev_center)
+of a polytope.
+
+### Input
+
+- `P`          -- polytope
+- `get_radius` -- (optional; default: `false`) option to additionally return the
+                  radius of the largest ball enclosed by `P` around the
+                  Chebyshev center
+- `backend`    -- (optional, default: `default_polyhedra_backend(P, N)`) the
+                  backend for polyhedral computations
+
+### Output
+
+If `get_radius` is `false`, the result is the Chebyshev center of `P`.
+If `get_radius` is `true`, the result is the pair `(c, r)` where `c` is the
+Chebyshev center of `P` and `r` is the radius of the largest ball with center
+`c` enclosed by `P`.
+
+### Notes
+
+The Chebyshev center is the center of a largest Euclidean ball enclosed by `P`.
+In general, the center of such a ball is not unique (but the radius is).
+"""
+function chebyshev_center(P::AbstractPolyhedron{N};
+                          get_radius::Bool=false,
+                          backend=default_polyhedra_backend(P, N),
+                          solver=JuMP.with_optimizer(GLPK.Optimizer)
```

Comment:
we can use here the [default lp solver](https://github.com/JuliaReach/LazySets.jl/blob/20c26b94f65074f538011a5b1144258a0ea755b2/src/Interfaces/AbstractPolyhedron_functions.jl#L10) -- but only after doing the [change to MOI](https://github.com/JuliaReach/LazySets.jl/pull/1701/files#diff-5e796138f8958d1d7c93fd38b67eb60aR10)

Reply:
I know :wink:

Reply:
Actually, the answer is "no". `JuMP.with_optimizer(GLPK.Optimizer)` returns an `OptimizerFactory`:
```julia
julia> JuMP.with_optimizer(GLPK.Optimizer)
OptimizerFactory(GLPK.Optimizer, (), Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}())
```